<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>1113</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="box_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>1070</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>box2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="box_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>1070</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>box1(white)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_file">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>1070</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>NextFile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prev_file">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>1070</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>PrevFile</string>
   </property>
  </widget>
  <widget class="QLabel" name="filenamelabel">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>15</y>
     <width>140</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>filename</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="imagelabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>1440</width>
     <height>960</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>ImageVeiwer</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="boxlist">
   <property name="geometry">
    <rect>
     <x>1520</x>
     <y>50</y>
     <width>241</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="box_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>1070</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>box4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="box_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>1070</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>box3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="box_6">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>1070</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>box6(black)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="box_5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>1070</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>box5</string>
   </property>
  </widget>
  <widget class="QLabel" name="mousecord">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>1010</y>
     <width>601</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Press top left corner of white box</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>1070</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
